stix_shifter/src/modules/csa/json/to_stix_map.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
Does this mapping work with CloudSQL? This looks like the same mapping used for QRadar.